modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/removeProductFromMinicart.js

```diff
@@ -0,0 +1,9 @@
+import getProductFromMinicart from '~/cypress/support/pageObjects/minicart/getProductFromMinicart'
```

Comment:
This page object seems to be unused

Reply:
not used in test files, but used in page objects 
<img width="978" alt="image" src="https://user-images.githubusercontent.com/114248299/222691056-6d6db9a8-a16f-4c65-b8dd-7f52b917b7e5.png">